pkg/distro/fedora/imagetype.go

```diff
@@ -514,3 +517,19 @@ func (t *imageType) checkOptions(bp *blueprint.Blueprint, options distro.ImageOp
 
 	return warnings, nil
 }
+
+// XXX: this will become part of the yaml distro definitions, i.e.
+// the yaml will have a "bootstrap_ref" key for each distro/arch
+func bootstrapContainerFor(t *imageType) string {
+	arch := t.arch.Name()
+	distro := t.arch.distro
+
+	// XXX: remove once fedora containers are part of the upstream
+	// fedora registry (and can be validated via tls)
```

Comment:
Is there an upstream tracking ticket for this? I'm not *super* happy making your personal account load bearing even if it's just for `riscv64`.

Reply:
I created https://github.com/osbuild/images/issues/1351 for us, we could move the repo to osbuild for now, its just a bunch of busywork - my hope was that we relatively soon see riscv in the upstream registry but maybe I'm too optimistic here.